### PR TITLE
Add STARE spatiotemporal encoding library API for geodata

### PR DESCRIPTION
**Pull request by michaelleerilee**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [n/a] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ok] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/stare/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  sha256: 783b53dfa93b7214d34fd78c759b34e84adac8aa9266d3da5e73704708adca5e
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
```
Comment (chrisburr) [inline recipes/stare/meta.yaml:20]:
```suggestion
    - cmake
    - make
```

File: recipes/stare/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libSTARE.a  # [not win]
```
Comment (chrisburr) [inline recipes/stare/meta.yaml:24]: Can this be a shared library instead?

File: recipes/stare/stare-activate.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+export STARE_INCLUDE_DIR=${CONDA_PREFIX}/include/STARE
+export STARE_LIB_DIR=${CONDA_PREFIX}/lib
```
Comment (chrisburr) [inline recipes/stare/stare-activate.sh:4]: Are these needed? Ideally the code should be compiled to default to these values (they can be hard coded in the binary and conda will fix the to the correct path at install time)

Comment (michaelleerilee) [reply recipes/stare/stare-activate.sh:4]: Other packages (e.g. on PyPi) currently use these during their build. Is there a better approach?

Comment (chrisburr) [reply recipes/stare/stare-activate.sh:4]: Ah you're wanting these for downstream dependencies to be able to build against? If so you probably don't need to do anything when using conda-forge's compilers as these are the default locations.

File: recipes/stare/meta.yaml (hunk 2 above)
Comment (michaelleerilee) [reply recipes/stare/meta.yaml]: Yes. A shared library is now generated.

File: recipes/stare/stare-activate.sh (hunk 3 above)
Comment (michaelleerilee) [reply recipes/stare/stare-activate.sh]: I've removed these. Since we're in a conda-forge environment, they should be easier to find.

**Comment by michaelleerilee** [comment]
@conda-forge-admin, please ping team

**Comment by michaelleerilee** [comment]
@conda-forge-admin, please ping team

**Review by michaelleerilee — COMMENTED** [review]
Seems okay to me, but then I wrote them.

**Review by michaelleerilee — COMMENTED** [review]
The activate/deactivate scripts are superfluous and have been deleted.

**Comment by michaelleerilee** [comment]
I know we all get busy, but I don't know what the hold-up is.

File: recipes/stare/build.sh [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+cmake -S $SRC_DIR \
```
Comment (isuruf) [inline recipes/stare/build.sh:3]:
```suggestion
cmake ${CMAKE_ARGS} -S $SRC_DIR \
```

File: recipes/stare/build.sh [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+      -DBUILD_SHARED_LIBS=YES \
+      -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+      -DSTARE_INSTALL_LIBDIR=lib
+
+make
```
Comment (isuruf) [inline recipes/stare/build.sh:8]:
```suggestion
make -j${CPU_COUNT}
```

**Review by isuruf — APPROVED** [review]

**Comment by michaelleerilee** [comment]
@conda-forge-admin, please ping team

**Comment by michaelleerilee** [comment]
So what's next?

**Pull request merged**